docs/assessment/index.md

```diff
@@ -22,35 +22,60 @@ Each assessment is a single directory in the `assessments` folder or any subfold
+| ----------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `uuid`                  | string  | Unique identifier (UUID). E.g., `"8b4891d6-64d1-4e89-b72d-ad2133f25b2f"`. These can be obtained from [uuidgenerator.net](https://www.uuidgenerator.net) (Required; no default) |
+| `type`                  | string  | Either `"Homework"` or `"Exam"`. ([Details;](#assessment-types) Required; no default)                                                                                          |
+| `title`                 | string  | The title of the assessment (e.g., `"Derivatives and integrals"`). (Required; no default)                                                                                      |
+| `set`                   | string  | Which assessment set this belongs to (e.g., `"Quiz"`, `"Practice Quiz"`). (Required; no default)                                                                               |
+| `number`                | string  | The number of the assessment within the set (e.g., `"1"`, `"2B"`). (Required; no default)                                                                                      |
+| `module`                | string  | The module that this assessment belongs to (e.g., `Chapter 3`). ([Details;](#grouping-assessments-by-modules) Optional; default: none)                                         |
+| `allowAccess`           | array   | List of access rules. ([Details;](../accessControl/index.md) Optional; default: no student access)                                                                             |
+| `zones`                 | array   | Specification of zones and questions. ([Details;](#question-specification) Optional; default: none)                                                                            |
+| `text`                  | string  | HTML text shown on the assessment overview page. (Optional; default: none)                                                                                                     |
+| `multipleInstance`      | boolean | Whether to allow students to create whole new attempts at the entire assessment. (Optional; default: `false`)                                                                  |
+| `maxPoints`             | number  | The maximum points that can be earned. (Optional; default: sum of zone max points)                                                                                             |
+| `maxBonusPoints`        | number  | The maximum number of additional points that can be earned beyond `maxPoints`. (Optional; default: 0)                                                                          |
```

Comment:
Do we have text to link to that talks about this?

Reply:
Done in bbb2e70.